x-pack/plugins/security_solution/common/detection_engine/schemas/alerts/8.3.0/index.ts

```diff
@@ -54,3 +51,17 @@ export interface EqlBuildingBlockFields830 extends BaseFields830 {
   [ALERT_GROUP_INDEX]: number;
   [ALERT_BUILDING_BLOCK_TYPE]: 'default';
 }
+
+export interface NewTermsFields830 extends BaseFields830 {
+  [ALERT_NEW_TERMS]: Array<string | number | null>;
```

Comment:
`Array<string | number | null>` - are we explicitly normalizing terms fetched from ES to `string | number | null` somewhere in the code?

Reply:
`string | number | null` are the only types that can come back as values in the `composite` aggregation - the type is seen on `const bucketsForField = searchResultWithAggs.aggregations.new_terms.buckets;` where we extract the buckets from the composite agg.

Reply:
Gotcha!